docs/backup_troubleshooting.md

```diff
@@ -49,3 +50,9 @@ This message appears when the previous replicated VM has been deleted on the tar
 This message appears when you try to do a backup/snapshot from a VM that was previously on a host with an active commercial XenServer license and which is now on a host with a free edition of XenServer.
 
 To solve it, you have to change a parameter in your VM. `xe vm-param-set has-vendor-device=false uuid=<VM_UUID>`
+
+### ENOSPC: no space left on device
+
+This message appears when you do not have enough free space when running a backup.  
```

Comment:
Try to explain where this space is lacking: it's not on the SR itself, but on the remote.

Reply:
done!